components/org.wso2.carbon.identity.application.authenticator.facebook/src/main/java/org/wso2/carbon/identity/application/authenticator/facebook/FacebookAuthenticator.java

```diff
@@ -415,6 +424,23 @@ protected String getLoginType(HttpServletRequest request) {
     }
 
     @Override
+    public String getClaimDialectURI() {
+        return FacebookAuthenticatorConstants.CLAIM_DIALECT_URI;
+    }
+
+    protected ClaimMapping buildClaimMapping(String claimUri) {
+        if (log.isDebugEnabled()) {
+            log.debug("Adding claim mapping" + claimUri);
+        }
+        ClaimMapping claimMapping = new ClaimMapping();
+        Claim claim = new Claim();
+        claim.setClaimUri(FacebookAuthenticatorConstants.CLAIM_DIALECT_URI + "/" + claimUri);
```

Comment:
Instead of using FacebookAuthenticatorConstants.CLAIM_DIALECT_URI better if we can use the newly defined getClaimDialectURI() method. 
pretty much the same :)